### ISSUE TITLE
OSX + Hostmanager not working

### ISSUE THREAD

**Issue by higidi** [puphpet/puphpet#2311]
On OSX the Hostmanager didn't work anymore? Anyone the same problems?

**Comment by EofChris** [comment]
I run into the same problem here. Maybe https://github.com/devopsgroup-io/vagrant-hostmanager/issues/179 gives a hint what the problem might be?

> I figured this out by trial and error and I would love someone confirm: config.hostmanager.enabled = true will work only if configured at the topmost level.

> So this will work: config.hostmanager.enabled = true

> And this will not: somenode.hostmanager.enabled = true

**Comment by higidi** [comment]
Yes, this fix the issue!

**Comment by mobyjames** [comment]
I modified my Vagrantfile so it looks like the following and this fixed the issue, but resulted in a duplicate entry in the host file (which is better than no entries I guess).

```
Vagrant.configure('2') do |config|
  config.hostmanager.enabled = true
  eval File.read("#{dir}/puphpet/vagrant/Vagrantfile-#{data['target']}")
end
```

**Comment by ikari7789** [comment]
This should be fixed if #2410 is merged in. @mobyjames thanks for that snippet.
